patterns/bottlerocket/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
```

Comment:
I can't comment on the empty file, but just above we can remove the `variables.tf` - we don't have modules her so we don't enforce the standard module convention anymore